system-event-service/src/main/resources/system-event-service-openapi.yaml

```diff
@@ -0,0 +1,3662 @@
+      properties:
+        items:
+          type: array
+          items:
+            $ref: '#/components/schemas/SystemEventRecord'
+        context:
+          $ref: '#/components/schemas/PaginationContextSystemEvent'
+ 
+##-------------------------------------------------------------------------
+security:
+ - tip_wlan_ts_auth: []
+paths:
+    
+  /api/systemEvent:
+    post:
+      tags:
+      - System Event Data
+      summary: Creates new SystemEvent
+      operationId: createSystemEvent
+      requestBody:
+        description: SystemEvent Record
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/SystemEventRecord'
+      responses:
+        200:
+          description: Successful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/GenericResponse'
+        500: 
+          $ref: '#/components/responses/GenericApiError'
+    delete:
+      tags:
+      - System Event Data
+      summary: Deletes service metric record
+      operationId: deleteServiceMetricRecord
+      parameters:
+      - name: customerId
+        in: query
+        description: Customer ID
+        required: true
+        schema:
+          type: integer
+          format: int32
+      - name: equipmentId
+        in: query
+        description: Equipment ID
+        required: true
+        schema:
+          type: integer
+          format: int64
+      - name: createdBeforeTimestamp
+        in: query
+        description: Created before timestamp
+        required: true
+        schema:
+          type: integer
+          format: int64
+      responses:
+        200:
+          description: Successful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/GenericResponse'
+        500: 
+          $ref: '#/components/responses/GenericApiError'
+  
+  /api/systemEvent/bulk:
+    post:
+      tags:
+      - System Event Data
+      summary: Creates batch of SystemEventRecords
+      operationId: createBulkSystemEventRecord
+      requestBody:
+        description: System Event Records
+        required: true
+        content:
+          application/json:
+            schema:
+              type: array
+              items:
+                $ref: '#/components/schemas/SystemEventRecord'
+      responses:
+        200:
+          description: Successful operation
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/GenericResponse'
+        500: 
+          $ref: '#/components/responses/GenericApiError'
+          
+  /api/systemEvent/forCustomer:  
+    get:
+      tags:
+      - System Event Data
+      summary: Gets service metric record for customer
+      operationId: getServiceMetricRecordForCustomer
```

Comment:
This API is about System Events, not Service Metrics - operationId and summary should reflect that